sematic/tests/test_api_client.py

```diff
@@ -147,3 +154,24 @@ def test_notify_resilient(mock_requests):
     _notify_event("foo", "bar", {})
 
     mock_requests.post.assert_called()
+
+
+@mock.patch("sematic.api_client._notify_event")
+def test_save_metrics_points(
+    mock_notify_event: mock.MagicMock,
+    metric_points: List[MetricPoint],  # noqa: F811
+    test_db: DB,  # noqa: F811
+    mock_requests_fixture,  # noqa: F811
+):
+    save_metric_points(metric_points)
+
+    payload = dict(
```

Comment:
Currently the testing code duplicates the exact functionality of the tested code. Testing code should be less complex, have a different implementation, and preferably be explicit.
Please manually construct a payload fixture that corresponds to the `metric_points` fixture, and import it here instead.